js/task17.js

```diff
@@ -0,0 +1,129 @@
+const ul = document.getElementById('lists');
+
+let currentImage = 0;
+const imagesArr = [];
+
+function createLoader() {
+	const fragment = document.createDocumentFragment();
+	const loaderContainer = document.createElement('li');
+	const loader = document.createElement('img');
+	loader.src = './img/loading-circle.gif';
+	fragment.appendChild(loader);
+	loader.appendChild(loaderContainer);
+	loader.classList.add('loader_image');
+	return loader;
+}
+
+function fetchData() {
+	return new Promise((resolve, reject) => {
+		setTimeout(() => {
+			resolve(fetch('./js/json/task17.json'));
+		}, 3000);
+	});
+}
+
+async function getData() {
+	const loaderImage = createLoader();
+
+	const fragment = document.createDocumentFragment();

```

Comment:
I think you can write without Fragment,
and you can write more concisely.

Reply:
Got it.